Userland/Services/WindowServer/ConnectionFromClient.cpp

```diff
@@ -976,6 +976,20 @@ Messages::WindowServer::GetDoubleClickSpeedResponse ConnectionFromClient::get_do
     return WindowManager::the().double_click_speed();
 }
 
+void ConnectionFromClient::set_mouse_size(int size)
```

Comment:
Can we name it maybe `set_cursor_size` to keep consistency? After all, we're changing the cursor's size, not the mouse's.